### ISSUE TITLE
Docs?

### ISSUE THREAD

**Issue by Boscop** [carllerche/kabuki#3]
Nothing here: https://docs.rs/kabuki/0.0.1/kabuki/

**Comment by carllerche** [comment]
Yeah, I haven't published yet... Kabuki is still at the "initial idea hack"...
